SyphonCFMessageSender.m

```diff
@@ -58,11 +58,13 @@ - (id)initForName:(NSString *)name protocol:(NSString *)protocolName invalidatio
         _queue.userInfo = (__bridge void *)(self);
 		// local vars for block references, see note below
 		SyphonMessageQueue *queue = _queue;
+		__weak SyphonCFMessageSender *weakSelf = self;
 		_dispatch = SyphonDispatchSourceCreate(^(){
-			
 			//// IMPORTANT																					//
 			//// Do not refer to any ivars in this block, or self will be retained, causing a retain-loop	//
-			
+			SyphonCFMessageSender *blockSafeSelf = weakSelf;
+			if (blockSafeSelf == nil)
```

Comment:
The call to CFMessagePortSendRequest should proceed in this case (port and queue are both retained by the block). Delete this check and the early return.